src/AspNetCoreModuleV2/CommonLib/iapplication.h

```diff
@@ -8,12 +8,8 @@
 
 enum APPLICATION_STATUS
```

Comment:
Why not just make this a bool like `IsApplicaitonRunning`?

Reply:
I was thinking about it but want to leave it as enum for future proofing.